lambda-packages/role_handler/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     '<rootDir>/test'
   ],
   transform: {
-    '^.+\\.tsx?$': 'ts-jest'
+    // '^.+\\.tsx?$': 'ts-jest'
```

Comment:
did you mean to comment this out?

Reply:
yes, I commented this out as this requires additional dependencies (typescript packages), which are not in the package.json